extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -104,35 +120,24 @@ public void build(ApplicationInfoBuildItem applicationInfo,
             throw new RuntimeException("Unable to setup environment for generating Kubernetes resources", e);
         }
 
-        Config config = ConfigProvider.getConfig();
-        List<String> deploymentTargets = Arrays
-                .stream(config.getOptionalValue(DEPLOYMENT_TARGET, String.class)
-                        .orElse(KUBERNETES).split(","))
-                .map(String::trim)
-                .map(String::toLowerCase)
-                .collect(Collectors.toList());
+        Map<String, Object> config = KubernetesConfigUtil.toMap();
+        config.entrySet().forEach(e -> System.out.println(e.getKey() + "=" + e.getValue()));
 
-        Map<String, Object> configAsMap = StreamSupport.stream(config.getPropertyNames().spliterator(), false)
-                .filter(k -> ALLOWED_GENERATORS.contains(generatorName(k)))
```

Comment:
With the references to `ALLOWED_GENERATORS` now gone, I think the field is no longer used.